Schedule-API/Schedule-API/Models/Event.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ScheduleAPI.Models
+{
+    public partial class Event
+    {
+        public int Id { get; set; }
+        public string Type { get; set; }
```

Comment:
@elekamate this `Type` could be an `enum` so everyone knows what types we have. Also it is not a mistake, just something that could make it cleaner.